docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to use the alpha commands to provision a cluster, install and upgrade Kyma, and handle errors.
+
+## Provision a cluster
+To provision a Kubernetes cluster, run:
+
+```
+kyma alpha provision k3s 
+```
+If you want to define the name of your cluster and pass arguments to the Kubernetes API server (for example, to log to standard errors), run:
+
+```
+kyma alpha provision k3s --name='custom_name' --server-args='--alsologtostderr'
+```
+
+
+## Install Kyma
+
+There are several ways to install Kyma:
+
+- To install Kyma on a local cluster, simply use the `deploy` command without any flags.
```

Comment:
That is not specific to a local cluster. You always can just call kyma deploy to any kubernetes cluster. The rest is advanced :)
Just the default URL is specific to a local setup. When calling it on a gardener cluster, then the default URL will be different and cluster specific.

Reply:
```suggestion
- To install Kyma, you can simply use the `deploy` command without any flags.
```